roles/foreman_proxy_certs_generate/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Set tarball path
+  set_fact:
+    foreman_proxy_certs_generate_tarball: /root/{{ foreman_proxy_certs_generate_fqdn }}.tar.gz
+  when: foreman_proxy_certs_generate_tarball is undefined
+
+- name: 'Generate certificates for a Foreman Proxy'
+  command: |
+    foreman-proxy-certs-generate
```

Comment:
should the name of the command be overrideable?

Reply:
Depends who the stakeholder is :P But I can fix that. I was just so excited to get this working.